components/reference-implementation-openbanking-consumerdatastandards-au/src/main/java/org/wso2/openbanking/consumerdatastandards/au/extensions/constants/CDSErrorEnum.java

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.openbanking.consumerdatastandards.au.extensions.constants;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Error enumerations for CDS Specific Errors.
+ */
+public enum CDSErrorEnum {
+    /**
+     * Error enumerations defined in the specification.
+     * <p>
+     * Custom values in the enum will be in the following order
+     * (HTTP Status Code, Error Code, Error Code Title, Error Message)
+     */
+    //Query or Request Body Parameter missing
+    FIELD_MISSING(400, "urn:au-cds:error:cds-all:Field/Missing", "Missing Required Field",
+            "Missing Required Field %s in the request"),
+    //one or more http headers are missing
+    HEADER_MISSING(400, "urn:au-cds:error:cds-all:Header/Missing", "Missing Required Header",
+            "Mandatory header %s is missing"),
+    //Query or Request Body Parameter is not defined by the API
+    UNEXPECTED_FIELD(400, "AU.CDR.Unexpected.Field", "Unexpected Field Not Allowed", "Unexpected Field %s is " +
+            "not allowed by the API"),
+    //Header is not defined by the API
+    UNEXPECTED_HEADER(400, "AU.CDR.Unexpected.Header", "Unexpected Header Not Allowed", "Unexpected Header " +
+            "Not Allowed"),
+    //Query, PAth or Request Body Parameter is invalid
```

Comment:
```suggestion
    //Query, Path or Request Body Parameter is invalid
```

Reply:
Fixed in [ab3cd77](https://github.com/wso2/reference-implementation-consumerdatastandards-au/pull/132/commits/ab3cd7712162cfa264437664bd0b42d28ff6034b)